spec/lib/miq_expression/tag_spec.rb

```diff
@@ -157,8 +157,12 @@
   end
 
   describe "#attribute_supported_by_sql?" do
-    it "is always false" do
-      expect(described_class.new(Vm, [], "host")).not_to be_attribute_supported_by_sql
+    it "is supported by sql" do
+      expect(described_class.new(Vm, [], "host")).to be_attribute_supported_by_sql
```

Comment:
this was my original question for libor, and he showed that indeed, queries with tags are supported by sql.
so this change looks good to me